src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs

```diff
@@ -84,68 +82,92 @@ public async Task<bool> PushMetadataAsync(CancellationToken cancellationToken)
                 }
                 else
                 {
-                    (List<BuildData> buildsManifestMetadata,
-                     List<SigningInformation> signingInformation,
-                     ManifestBuildData manifestBuildData) = GetBuildManifestsMetadata(ManifestsPath, cancellationToken);
+                    //get the list of manifests
+                    List<Manifest> parsedManifest = GetParsedManifest(ManifestsPath, cancellationToken);
```

Comment:
Nit: variable could be `parsedManifests` and the method could be `GetParsedManifests` or `ParseManifests`?

Reply:
done